pkg/graph/resolvers/system_intake_document.go

```diff
@@ -226,3 +239,33 @@ func canDelete(ctx context.Context, store *storage.Store, id uuid.UUID) error {
 
 	return errors.New("unauthorized attempt to delete system intake document")
 }
+
+func handleSendEmail(ctx context.Context, store *storage.Store, emailClient *email.Client, intake *models.SystemIntake) error {
+	grbUsers, err := store.SystemIntakeGRBReviewersBySystemIntakeIDs(ctx, []uuid.UUID{intake.ID})
+	if err != nil {
+		return err
+	}
+
+	ids := make([]uuid.UUID, len(grbUsers))
+	for i := range ids {
+		ids[i] = grbUsers[i].UserID
+	}
+
+	accounts, err := store.UserAccountsByIDs(ctx, ids)
```

Comment:
Is there any downside to using the GetUserAccount dataloader instead of the store method directly?

https://github.com/CMS-Enterprise/easi-app/blob/a81044d3fd221937e3b080f360ffb0bab6fab4b6/pkg/dataloaders/dataloaders.go#L96

Reply:
I would think only in the test files in that you have to use the context helper to reload the dataloaders. In production, I don't think there's a big difference.

Reply:
@ClayBenson94 i agree! will update

@mynar7 is this how that would be done? for some reason this doesn't seem right 😬 

something like 
```suggestion
	var accounts []*authentication.UserAccount
	for _, id := range ids {
		account, err := dataloaders.GetUserAccountByID(ctx, id)
		if err != nil {
			return err
		}

		accounts = append(accounts, account)
	}
```

Reply:
i feel like i am forgetting the basics 😂 i should build out a GetUserAccountsByID (plural accounts) for this right

Reply:
Looping to call the dataloader just feels so wrong. For a mutation I don't think it's worth worrying about the dataloader to be honest, I just didn't see it as a negative until now.